pyipn/detector.py

```diff
@@ -1,10 +1,14 @@
 import numpy as np
 import astropy.constants as constants
 
-
-from .lightcurve import LightCurve
-from .possion_gen import source_poisson_generator, background_poisson_generator
-
+try:
+    from .lightcurve import LightCurve
```

Comment:
Why do we need these import checks?

Reply:
I couldn't get these relative imports to run locally. That's an issue on my end but after almost an hour of unsuccessful troubleshooting I settled for this solution.